github-profiles/YovanniPortilla-5475/main.css

```
@@ -0,0 +1,30 @@
+body{
+    margin: 0 auto;
+    max-width: 50em;
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 1.5;
+    padding: 4em 1em;
+    color: #555;
+}
+
```

El trabajo está bien pero te recomendaría que en el css agregaras el siguiente código para arreglar la imagen
```suggestion
img {
    width: 100%;
}
```